azure-mobile-apps/azure-mobile-apps.d.ts

```diff
@@ -27,11 +27,11 @@ declare module "azure-mobile-apps/src/query" {
     export = query;
 }
 
-declare namespace Azure.MobileApps {
+declare module Azure.MobileApps {
```

Comment:
`namespace` is actually preferred. see http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html